extensions/ql-vscode/docs/test-plan.md

```diff
@@ -0,0 +1,257 @@
+# Test Plan
+
+This document describes the manual test plan for the QL extension for Visual Studio Code.
+
+The plan will be executed manually to start with but the goal is to eventually automate parts of the process (based on 
+effort vs value basis).
+
+#### What this doesn't cover
+We don't need to test features (and permutations of features) that are covered by automated tests.
+
+#### Areas to consider
+1. Running MRVA and viewing results
+2. Running local queries
+3. Interacting with the query history
+4. Exporting MRVA results
+5. Cancelling a MRVA run
+6. Changing colour theme
+
+### Before releasing the VS Code extension
+- Check major PRs since the previous release for specific one-off things to test.
+- Run a query using the existing version of the extension (to generate an “old” query history item)
+- Go through the required test cases listed below
+
+## Required Test Cases
+
+### Test Case 1: Running a problem path query and viewing results
+
+1. Open the [UnsafeJQueryPlugin query](https://github.com/github/codeql/blob/main/javascript/ql/src/Security/CWE-079/UnsafeJQueryPlugin.ql).
+2. Run it against the following repo list:
```

Comment:
Perhaps it's worth clarifying that we should run a variant analysis here (even that may be obvious from the mention of a repo list)?